index.html

```diff
@@ -90,10 +90,22 @@ <h3>
       <dfn>RTCAudioReceiverStats</dfn> dictionary non-standardized members
     </h3>
     <pre class="idl">partial dictionary RTCAudioReceiverStats {
+      unsigned long long delayedPacketOutageSamples;
       unsigned long long jitterBufferFlushes;
 };</pre>
     <dl data-link-for="RTCAudioReceiverStats" data-dfn-for="RTCAudioReceiverStats" class=
     "dictionary-members">
+      <dt>
+        <dfn><code>delayedPacketOutageSamples</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which is available <a href="https://docs.google.com/document/d/1stYIZhEmDZ7NJF9gjjsM66eLFJUdc-14a3QutrFbIwI/edit#" title="experiment description">under origin trial in Chromium starting from M72 version</a>.
+          Measures the number of audio samples that were concealed due to jitter buffer underrun caused by delayed packets.
```

Comment:
We have concealment metrics already defined.

This proposal seems to create a nuanced version of those concealment metrics. So my question is what does this metric provide input to.

Reply:
Yes, this metric is a subset of concealedSamples. The metric provide input into how well the jitter buffer's network adaptation is working. Since concealment caused by packet loss can't be prevented by the jitter buffer, this metric gives a more accurate number of what can be improved. It can also give a ratio of how much concealment is caused by packet loss and how much is caused by jitter, by dividing with concealedSamples.

Reply:
@henbos